### PR TITLE
Updated schedule for HOTH X

### LINKED ISSUE #306: Update schedule
Update schedule on Schedule page to have all timestamps as provided by HOTH directors, and add a short description of the event as well!

Schedule is in HOTH X folder on Google Drive under Day of Logistics. Only put the events under Participant Events on the Schedule page.

### PR DESCRIPTION
**Pull request by maggieelli**
Fixes #306

### CODE REVIEW

**Comment by nathanzzhang** [comment]
not eslint failing LMAO

File: src/data/eventSchedule.js [hunk 1]
```diff
@@ -54,53 +53,69 @@ const eventSchedule = [
...
 	{
-		startTime: getDateObj('26', '18:00:00'),
+		startTime: getDateObj('05', '18:00:00'),
 		events: [
 			{
 				name: 'Dinner',
 				duration: '60 min',
-				location: 'Covel Grand Horizon Terrace',
+				location: 'Carnesale Commons Palisades Room',
 				description: `Do you know what's better than one free meal? Two
```
Comment (nathanzzhang) [inline src/data/eventSchedule.js:119]: Wait aren't we providing them w 3 free meals 😭

File: src/data/eventSchedule.js [hunk 2]
```diff
@@ -109,45 +124,57 @@ const eventSchedule = [
...
 		]
 	},
 	{
-		startTime: getDateObj('26', '22:30:00'),
+		startTime: getDateObj('05', '22:30:00'),
 		events: [
 			{
 				name: 'Closing Ceremony',
 				duration: '20 min',
 				location: 'Covel Grand Horizon/Zoom',
```
Comment (nathanzzhang) [inline src/data/eventSchedule.js:170]: Just want to make sure that we are actually having the closing ceremony on zoom as well

**Review by nathanzzhang — CHANGES_REQUESTED** [review]
some minor content questions

**Review by nathanzzhang — APPROVED** [review]
lgtm!

**Review by khxia — DISMISSED** [review]

File: src/data/eventSchedule.js [hunk 3]
```diff
@@ -54,54 +53,70 @@ const eventSchedule = [
...
-				description: `We'll be having a guided discussion about topics
-				related to the Netflix documentary 'The Social Dilemma'
-				and the ethics of technology from the morality of data collection
-				to the environmental impact of tech to algorithmic bias in AI.
-				Get a raffle entry just for attending and a second one for participating.
-				Sounds like a bargain, no?`
+				name: 'Interdisciplinary Research Panel',
+				duration: '60 min',
+				location: 'Carnesale Commons Palisades Room',
+				description: `We have an amazing panel of UCLA professors/researches whose
```
Comment (khxia) [inline src/data/eventSchedule.js:104]: Wait is it supposed to be 'professors/researches' or 'professors/researchers'?

Comment (maggieelli) [reply src/data/eventSchedule.js]: oops it's supposed to be researchers, just fixed

**Review by khxia — APPROVED** [review]
辛苦晒~!

**Pull request merged**